capstone/src/main/java/com/google/step/data/Restaurant.java

```diff
@@ -79,7 +75,7 @@ public String getStory() {
    * @return List<String> cuisine, list of cuisines for the restaurant
    */
   public List<String> getCuisine() {
-    return new ArrayList<>(this.cuisine);
+    return Collections.unmodifiableList(cuisine);
```

Comment:
Do this in the constructor instead of here, then you can just return it here.

Here's why. Consider this code:

```
List<String> cuisine = Arrays.asList("italian", "french");
Restaurant r = new Restaurant(<...>, cuisine, <...>);
cuisine.add("poop");

assertFalse(r.getCuisine().contains("poop"));
```

To make sure nobody with access to the original list can add "poop" to your cuisine after your restaurant has been created, you need to make sure the list owned by Restaurant is its own unmodifiable list.

Reply:
Ah, now I see. Thank you!!